### DISCUSSION TITLE
auditee_ein missing from federal_awards (formerly CFDA)

### DISCUSSION THREAD

**Discussion by benjamin-fleming** [GSA-TTS/FAC#3542] [General]
I am trying to update and replace some CFDA reports that our agency created from the prior Census Bureau data however, I am having some trouble with the current federal_awards (formerly cfda) data as it does not include the auditee_ein even though it is listed as being included in the FAC Data Dictionary: https://www.fac.gov/api/dictionary/. Will the EIN be added at a later time?

**Comment by jadudm** [comment]
Hi @benjamin-fleming ,

It's in the `general` table. We don't, at this point, have an intention of duplicating it into another table.

Typically, when I'm working against the API, I find myself using the `report_id` (because it is a PK in `general` and `FK` everywhere else), to get what I need from any given table. In your case, you'll have to use the `report_id` against `general` to extract the `auditee_ein`. You can use `select` as part of a PostgREST query to slim down the amount of data you're slinging back and forth, if you want, and just get that field from the API.

https://postgrest.org/en/v12/references/api/tables_views.html#vertical-filtering

That might be your best bet right now. 

We might, in the future, have some differently-shaped tables, which would make this easier. For the moment, we have an API modeled on the old dissemination tables from Census.

Please feel free to ask more questions as needed.

**Reply by benjamin-fleming** [reply]
Sounds good - I’ll just add a join to our import process. Thank you for such a quick reply!
